### PR TITLE
显示充值数额，付费用户数量，审计封禁原因

### PR DESCRIPTION
**Pull request by regendsoh**
极为简易的知道自己到底赚了多少钱

### CODE REVIEW

**Comment by regendsoh** [comment]
继续坐等大佬们 @Anankke

File: resources/views/material/admin/code/index.tpl [hunk 1]
```diff
@@ -12,15 +12,20 @@
...
-                            <p>显示表项:
-                                {include file='table/checkbox.tpl'}
-                            </p>
-                        </div>
-                    </div>
-                </div>
+					<div class="card-main">
+						<div class="card-inner">
+							<p>系统中金额流转记录。</p>
+							<p><b>今日流水：￥{$user->tdyincome()}
```
Comment (SukkaW) [inline resources/views/material/admin/code/index.tpl]:
```suggestion
							<p>今日流水：￥{$user->tdyincome()}
```

File: resources/views/material/admin/code/index.tpl [hunk 2]
```diff
@@ -12,15 +12,20 @@
...
+							<br/>昨日流水：￥{$user->ytdincome()}
+							<br/>这月流水：￥{$user->mthincome()}
+							<br/>上月流水：￥{$user->lstmthincome()}
+							<br/>总流水：￥{$user->ttlincome()}</b></p>
```
Comment (SukkaW) [inline resources/views/material/admin/code/index.tpl]:
```suggestion
							<br/>总流水：￥{$user->ttlincome()}</p>
```

不建议使用 `<b>` 加粗、干脆去掉

Comment (regendsoh) [reply resources/views/material/admin/code/index.tpl]: 好der

**Comment by regendsoh** [comment]
本想放三个PR 奈何对github revert commit不熟悉 直接一起提交吧

**Comment by SakuraSa233** [comment]
I think this pull request shouldn't be merge due to:
1. bad code quality
2. bad commit message
4. bad pull request message

@regendsoh You should refactor your code by what [CONTRIBUTING.md](https://github.com/Anankke/SSPanel-Uim/blob/dev/CONTRIBUTING.md) writes and open a pull request again.

**Comment by regendsoh** [comment]
closed for recommit

> I think this pull request shouldn't be merge due to:
> 
> 1. bad code quality
> 2. bad commit message
> 3. bad pull request message
> 
> @regendsoh You should refactor your code by what [CONTRIBUTING.md](https://github.com/Anankke/SSPanel-Uim/blob/dev/CONTRIBUTING.md) writes and open a pull request again.

**Pull request closed without merging**